### DISCUSSION TITLE
How can I check if a type is a ref struct at runtime in netstandard2.0?

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/runtime#78566] [Q&A]
I need to check if `Type.IsByRefLike` is true. Problem is this property is not available in netstandard2.0.
I tried using a few nuget packages like System.Reflection, System.Reflection.Extensions, and System.Reflection.TypeExtensions. None of them worked.
I also tried looking for the presence of a `IsByRefLikeAttribute`, but I also can't find the NuGet package for it.
Any help?

**Comment by hez2010** [comment]
I'm afraid that you can't. ref struct didn't exist at the time netstandard2.0 was introduced. 
But checking the attribute `IsByRefLikeAttribute` on the type may be a workaround.
